### PR TITLE
Unstable tests

### PR DESCRIPTION
**Pull request by sebbi08**
# Readiness checklist

-   [X] I added/updated tests.
-   [ ] I ensured that the PR title is good enough for the changelog.
-   [X] I labeled the PR.

### CODE REVIEW

**Comment by jkoenig134** [comment]
@sebbi08 why is this no draft? and where is the WIP label? :)

**Comment by sebbi08** [comment]
No WIP / Draft as we need the pipelines to run

**Comment by jkoenig134** [comment]
> No WIP / Draft as we need the pipelines to run

pipelines run for Drafts and also when the WIP label is set.

-> Draft is for not confusing others with this PR, it makes your "DO NOT MERGE" unnecessary
-> WIP is to disable mergify, you do not need it for this PR

File: packages/runtime/test/transport/identityDeletionPeerProcessing.test.ts [hunk 1]
```diff
@@ -51,6 +51,8 @@ describe("IdentityDeletionProcess", () => {
 
     test("peer should be notified about cancelled deletion process", async function () {
         await services1.transport.identityDeletionProcesses.initiateIdentityDeletionProcess();
+        await syncUntilHasEvent(services2, PeerToBeDeletedEvent, (e) => e.data.id === relationshipId);
```
Comment (jkoenig134) [inline packages/runtime/test/transport/identityDeletionPeerProcessing.test.ts:54]: please add a comment, this should not be necessary and should be removed as soon the backbone resolves the core issue.

File: packages/runtime/test/transport/identityDeletionPeerProcessing.test.ts [hunk 2]
```diff
@@ -51,6 +51,7 @@ describe("IdentityDeletionProcess", () => {
 
     test("peer should be notified about cancelled deletion process", async function () {
         await services1.transport.identityDeletionProcesses.initiateIdentityDeletionProcess();
+        // This should not be necessary and can be removed once the backbone external events come in the right order
```
Comment (jkoenig134) [inline packages/runtime/test/transport/identityDeletionPeerProcessing.test.ts:54]:
```suggestion
        // This should not be necessary and can be removed once the backbone external events arrive in the right order
```

**Review by jkoenig134 — APPROVED** [review]

**Pull request merged**